baseapp-auth/baseapp_devices/utils.py

```diff
@@ -0,0 +1,74 @@
+import sys
+from hashlib import md5
+
+from django.conf import settings
+from django.core.cache import DEFAULT_CACHE_ALIAS
+from user_agents import parse
+
+
+# `get_cache` function has been deprecated since Django 1.7 in favor of `caches`.
```

Comment:
we only support django 4 or newer and python 3 or newer, should we care about this? the whole implementation is following old django style and supporting python 2